### PR TITLE
Adjust license key comment, update links in docs.

### PR DESCRIPTION
**Pull request by Mati365**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal: Adjust license key comment, update trial signup links in docs.

---

### Additional information

Closes https://github.com/cksource/ckeditor5-commercial/issues/6755

### CODE REVIEW

**Review by godai78 — DISMISSED** [review]
I cannae see no wrong with this boye.

File: packages/ckeditor5-core/src/editor/editorconfig.ts [hunk 1]
```diff
@@ -810,10 +810,11 @@ export interface EditorConfig {
...
 
 	/**
-	 * The license key for the CKEditor 5 commercial license and the premium features.
+	 * The CKEditor 5 license key. If you want to obtain a license key, please do one of the following:
 	 *
-	 * If you do not have a key yet, please [contact us](https://ckeditor.com/contact/) or
-	 * [order a trial](https://orders.ckeditor.com/trial/premium-features).
+	 * * [Contact us](https://ckeditor.com/contact/) for a commercial license.
+	 * * [Get a 14-day free trial](https://portal.ckeditor.com/checkout?plan=free) to test premium features.
+	 * * Use 'GPL' if you only need open source features.
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editorconfig.ts:817]: I need to tweak this a bit.

File: packages/ckeditor5-core/src/editor/editorconfig.ts [hunk 2]
```diff
@@ -810,10 +810,11 @@ export interface EditorConfig {
 	updateSourceElementOnDestroy?: boolean;
 
 	/**
-	 * The license key for the CKEditor 5 commercial license and the premium features.
+	 * The CKEditor 5 license key. If you want to obtain a license key, please do one of the following:
 	 *
-	 * If you do not have a key yet, please [contact us](https://ckeditor.com/contact/) or
-	 * [order a trial](https://orders.ckeditor.com/trial/premium-features).
+	 * * [Contact us](https://ckeditor.com/contact/) for a commercial license.
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editorconfig.ts:815]:
```suggestion
     * * Create a free account, and test the premium features with a [14-day free trial](https://portal.ckeditor.com/checkout?plan=free).
```

File: packages/ckeditor5-core/src/editor/editorconfig.ts [hunk 3]
```diff
@@ -810,10 +810,11 @@ export interface EditorConfig {
...
+	 * * [Get a 14-day free trial](https://portal.ckeditor.com/checkout?plan=free) to test premium features.
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editorconfig.ts:816]:
```suggestion
	 * * [Contact us](https://ckeditor.com/contact/) for a commercial license.
```

File: packages/ckeditor5-core/src/editor/editorconfig.ts (hunk 1 above)
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editorconfig.ts:817]:
```suggestion
	 * * If you are using the editor under a GPL license or another license from our Open Source Initiative,
	 * * use the 'GPL' license key instead.
```

**Review by Witoso — CHANGES_REQUESTED** [review]

File: packages/ckeditor5-core/src/editor/editorconfig.ts [hunk 4]
```diff
@@ -810,10 +810,12 @@ export interface EditorConfig {
...
 	/**
-	 * The license key for the CKEditor 5 commercial license and the premium features.
+	 * The CKEditor 5 license key. If you want to obtain a license key, please do one of the following:
 	 *
-	 * If you do not have a key yet, please [contact us](https://ckeditor.com/contact/) or
-	 * [order a trial](https://orders.ckeditor.com/trial/premium-features).
+	 * * Create a free account, and test the premium features with a [14-day free trial](https://portal.ckeditor.com/checkout?plan=free).
+	 * * [Contact us](https://ckeditor.com/contact/) for a commercial license.
+	 * * If you are using the editor under a GPL license or another license from our Open Source Initiative,
+	 *   use the 'GPL' license key instead.
```
Comment (Witoso) [inline packages/ckeditor5-core/src/editor/editorconfig.ts:818]: Not sure, should it be `* *` ?

Comment (Mati365) [reply packages/ckeditor5-core/src/editor/editorconfig.ts:818]: Nope, currently it looks like this:

obraz

but with `**` it looks like this:

obraz

**Review by Witoso — APPROVED** [review]

**Pull request merged**